docs/concepts/workloads/controllers/deployment.md

```diff
@@ -35,88 +35,109 @@ The following are typical use cases for Deployments:
 
 ## Creating a Deployment
 
-Here is an example Deployment. It creates a ReplicaSet to bring up three nginx Pods.
+The following is an example of a Deployment. It creates a ReplicaSet to bring up three `nginx` Pods:
 
 {% include code.html language="yaml" file="nginx-deployment.yaml" ghlink="/docs/concepts/workloads/controllers/nginx-deployment.yaml" %}
 
-Run the example by downloading the example file and then running this command:
+In this example:
+
+* A Deployment named `nginx` is created
```

Comment:
Just add periods to the bullets points. /lgtm otherwise

Reply:
Thanks! :)